src/StubQueue.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Queue;
+
+use Yiisoft\Queue\Adapter\AdapterInterface;
+use Yiisoft\Queue\Enum\JobStatus;
+use Yiisoft\Queue\Message\MessageInterface;
+use Yiisoft\Queue\Middleware\Push\MiddlewarePushInterface;
+
+/**
+ * Stub queue that does nothing. Job status is always "done".
```

Comment:
It's used in tests only, so let's move it to tests too. We've got tests/App directory for such stubs.